documentation/api/io.strimzi.api.kafka.model.Rack.adoc

```diff
@@ -0,0 +1,72 @@
+Configures rack awareness to spread partition replicas across different racks.
+
+A _rack_ can represent an availability zone, data center, or an actual rack in your data center.
+By configuring a `rack` for a Kafka cluster, consumers can fetch data from the closest replica.
+This is useful for reducing the load on your network when a Kafka cluster spans multiple datacenters.
+
+To configure Kafka brokers for rack awareness specify:
+
+* A `topologyKey` value to match the label of the cluster node used by Kubernetes when scheduling Kafka broker pods to nodes
+* The `RackAwareReplicaSelector` implementation for the Kafka `ReplicaSelector` plugin
```

Comment:
This is really related just to the client rackawareness / consuming from closest replica. Not to the broker rackawareness as suggested here. I think we need to make it clear.

Reply:
Changed to:

> To configure Kafka brokers for rack awareness, you can specify:
> 
> * A `topologyKey` value to match the label of the cluster node used by Kubernetes when scheduling Kafka broker pods to nodes
> * The `RackAwareReplicaSelector` implementation for the Kafka `ReplicaSelector` plugin if you want clients to consume from the closest replica

Reply:
Seeing the next comment, I'm going to move all the `RackAwareReplicaSelector` content to after the first example.